tests/ci/cdk/app.py

```diff
@@ -5,7 +5,7 @@
 
 from aws_cdk import core
 
-# from cdk.bm_framework_stack import BmFrameworkStack
```

Comment:
Do we want to leave this commented out like how we have things commented out [in lines 48-49](https://github.com/awslabs/aws-lc/pull/773/files#diff-81090b83c0407bfc08a3b3768f0781557eeabebe5397a96c78289149ae446067L48)? If no, then those should probably also be removed for consistency 😅